src/connectors/numeric-menu/__tests__/connectNumericMenu-test.ts

```diff
@@ -1127,66 +1127,6 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/numeric-men
     });
   });
 
-  describe('insights', () => {
```

Comment:
Could we maybe add a test asserting that no event is sent when adding a facet? This will make the behavior explicit and not look like an omission for a newcomer, or when we look back this part of the code in 6 months.

Reply:
Good idea, thank you for thinking of this!